### ISSUE TITLE
version 0.14.1 edit request transformer plugin problem

### ISSUE THREAD

**Issue by HFN** [pantsel/konga#347]
Looks like there is a problem with editing request transformer plugin in Konga 0.14.1 as you can see below
![konga_bug](https://user-images.githubusercontent.com/5382592/53692152-00438f00-3da0-11e9-916a-5064cc534141.png)
This will not happen when you are adding a new plugin. To reproduce the problem just add a new request transformer plugin and try to edit it.

**Comment by pantsel** [comment]
@HFN , what version of kong are you using?

**Comment by HFN** [comment]
Kong version 1.0.3

**Comment by arcana261** [comment]
Same problem here :+1:

**Comment by pantsel** [comment]
Guys, I cannot reproduce the problem.

![untitled-1](https://user-images.githubusercontent.com/5519873/53695407-1f541800-3dc4-11e9-9004-2a4dfc67220f.jpg)


What happens if you use Konga 0.14.2 or konga:latest,

Does the problem still persists?

**Comment by HFN** [comment]
I tried to investigate more on what is the problem and looks like i cannot reproduce it (with a clean new Kong and Konga) as well. I'm using Konga with Mongo database and for some reason when i'm using it with no database (I don't know whats the default when `DB_ADAPTER` is not set) its working perfectly. I tried recreating Mongo but that didn't solved it.

As i'm not sure anymore if it's something with Konga or there is another problem, I'm closing this issue for now. I will update this thread if i find anything.
